### ISSUE TITLE
logging inconsistencies with recent changes

### ISSUE THREAD

**Issue by inphobia** [netdisco/netdisco#479]
2 minor logging issuess combined in 1 report.

1. (#475) excellent that this got fixed, but i would like to know what's the way forward with:
   - imo logging got a bit inconsistant with this commit
   - only noticable when using netdisco-do i think
   - the old code reported ignored interfaces at debug level, the new code is reporting on info level
   - found with netdisco-do discover -DI
```
[12850] 2019-01-04 01:51:05 debug  [10.40.254.150] interfaces - ignoring 1004 (Po5) (notPresent) (config:ignore_notpresent_types)
[12850] 2019-01-04 01:51:05 debug  [10.40.254.150] interfaces - ignoring 1006 (Po7) (notPresent) (config:ignore_notpresent_types)

[12850] 2019-01-04 01:51:13  info  [10.40.254.150] properties/speed - local port Po7 not in database!
[12850] 2019-01-04 01:51:13  info  [10.40.254.150] properties/speed - local port Po5 not in database!

[12850] 2019-01-04 01:51:14  info  [10.40.254.150] vlans - local port Po7 not in database!
[12850] 2019-01-04 01:51:14  info  [10.40.254.150] vlans - local port Po5 not in database!
```

2. (#436) am seeing netdisco-do say some of my devices have manual topo while this is not the case. devices on the other end. devices on the other end are excluded from discovery due to not matching devices_only.

(note: it's not complaining about fa19 since that's a device matching discover_no_type)

```
[27342] 2019-01-07 12:41:12 debug  [10.40.254.150] neigh - setting manual topology links
[27342] 2019-01-07 12:41:14 debug  [10.40.254.150] neigh - 10.40.254.248 with ID [6c:b2:ae:91:25:0c] on fa1
[27342] 2019-01-07 12:41:14 debug  [10.40.254.150] neigh - 10.40.254.249 with ID [6c:b2:ae:91:22:be] on fa12
[27342] 2019-01-07 12:41:14 debug  [10.40.254.150] neigh - 10.40.254.111 with ID [44:d3:ca:bb:4a:00] on fa24
[27342] 2019-01-07 12:41:14  info  [10.40.254.150] neigh - fa1 has manually defined topology
[27342] 2019-01-07 12:41:14  info  [10.40.254.150] neigh - fa12 has manually defined topology
[27342] 2019-01-07 12:41:14 debug  [10.40.254.150] neigh - 10.40.254.102 with ID [netapp-cl2-01] on fa19
[27342] 2019-01-07 12:41:14  info  [10.40.254.150] neigh - fa24 has manually defined topology
[27342] 2019-01-07 12:41:14 debug is_discoverable: 10.40.254.248 failed to match discover_only
[27342] 2019-01-07 12:41:14 debug  queue - skip: 10.40.254.248 of type [Cisco Nexus Operating System (NX-OS) Software 7.0(3)I7(5)TAC support: http://www.cisco.com/tacCopyright (c) 2002-2018, Cisco Systems, Inc. All rights reserved] excluded by discover_* config
[27342] 2019-01-07 12:41:14 debug is_discoverable: 10.40.254.249 failed to match discover_only
[27342] 2019-01-07 12:41:14 debug  queue - skip: 10.40.254.249 of type [Cisco Nexus Operating System (NX-OS) Software 7.0(3)I7(5)TAC support: http://www.cisco.com/tacCopyright (c) 2002-2018, Cisco Systems, Inc. All rights reserved] excluded by discover_* config
[27342] 2019-01-07 12:41:14 debug is_discoverable: 10.40.254.111 failed to match discover_only
[27342] 2019-01-07 12:41:14 debug  queue - skip: 10.40.254.111 of type [Cisco IOS Software, C3750E Software (C3750E-UNIVERSALK9-M), Version 15.0(2)SE6, RELEASE SOFTWARE (fc2)Technical Support: http://www.cisco.com/techsupportCopyr] excluded by discover_* config
[27342] 2019-01-07 12:41:14 debug is_discoverable: 10.40.254.102 matched discover_no_type
[27342] 2019-01-07 12:41:14 debug  queue - skip: 10.40.254.102 of type [FAS8200] excluded by discover_* config
[27342] 2019-01-07 12:41:14 debug  [10.40.254.150] neigh - processed 4 neighbors


netdisco-do psql -e "select ip,port,manual_topo from device_port where ip = '10.40.254.248' order by port asc"
[26770] 2019-01-07 12:24:47  info App::Netdisco version 2.040002 loaded.
[26770] 2019-01-07 12:24:47  info psql:  started at Mon Jan  7 13:24:47 2019
 ip | port | manual_topo
----+------+-------------
(0 rows)

netdisco-do psql -e "select ip,port,remote_ip,manual_topo,is_uplink from device_port where ip = '10.40.254.150' order by port asc"
[26827] 2019-01-07 12:27:27  info App::Netdisco version 2.040002 loaded.
[26827] 2019-01-07 12:27:27  info psql:  started at Mon Jan  7 13:27:27 2019
      ip       |     port      |   remote_ip   | manual_topo | is_uplink
---------------+---------------+---------------+-------------+-----------
 10.40.254.150 | 1             |               | f           |
 10.40.254.150 | 254           |               | f           |
 10.40.254.150 | fa1           | 10.40.254.248 | f           | t
 10.40.254.150 | fa10          |               | f           |
 10.40.254.150 | fa11          |               | f           |
 10.40.254.150 | fa12          | 10.40.254.249 | f           | t
 10.40.254.150 | fa13          |               | f           |
 10.40.254.150 | fa14          |               | f           |
 10.40.254.150 | fa15          |               | f           |
 10.40.254.150 | fa16          |               | f           |
 10.40.254.150 | fa17          |               | f           |
 10.40.254.150 | fa18          |               | f           |
 10.40.254.150 | fa19          | 10.40.254.102 | f           | t
 10.40.254.150 | fa2           |               | f           |
 10.40.254.150 | fa20          |               | f           |
 10.40.254.150 | fa21          |               | f           |
 10.40.254.150 | fa22          |               | f           |
 10.40.254.150 | fa23          |               | f           |
 10.40.254.150 | fa24          | 10.40.254.111 | f           | t
 10.40.254.150 | fa3           |               | f           |
 10.40.254.150 | fa4           |               | f           |
 10.40.254.150 | fa5           |               | f           |
 10.40.254.150 | fa6           |               | f           |
 10.40.254.150 | fa7           |               | f           |
 10.40.254.150 | fa8           |               | f           |
 10.40.254.150 | fa9           |               | f           |
 10.40.254.150 | gi1           |               | f           |
 10.40.254.150 | gi2           |               | f           |
 10.40.254.150 | gi3           |               | f           |
 10.40.254.150 | gi4           |               | f           |
 10.40.254.150 | Logical-int 1 |               | f           |
 10.40.254.150 | loopback1     |               | f           |


[26735] 2019-01-07 12:22:41  info psql: finished at Mon Jan  7 13:22:41 2019
[26735] 2019-01-07 12:22:41  info psql: status done: psql session closed.

netdisco-do psql -e "select * from topology"
[26758] 2019-01-07 12:23:56  info App::Netdisco version 2.040002 loaded.
[26758] 2019-01-07 12:23:56  info psql:  started at Mon Jan  7 13:23:56 2019
 dev1 | port1 | dev2 | port2
------+-------+------+-------
(0 rows)
```

i'm only seeing this very recently but suspect it's related to #436 or #475.

tested on netdisco head.

if you let me know what the prefered logging verbosity level is i can change that for item 1. item 2 most likely needs a revisit of the sql.

**Comment by ollyg** [comment]
Thanks @inphobia 

Please do make a PR to change log to Debug level, and messages to "already skipped", as discussed.

I've pushed the following as an attempt to address the topology cache issue
[master 7f1074ff] #479 attempt better cache control for device ports

**Comment by inphobia** [comment]
98a5e2c should change info logging to debug logging.

**Comment by inphobia** [comment]
and manual topology spam fixed by 7f1074f
